pkg/fs/fs.go

```diff
@@ -25,3 +27,38 @@ type DiskStatus struct {
 	Free  uint64
 	Avail uint64
 }
+
+// MoveFileWithReplacement copies the file contents at `src` to `dst`.
+//
+// If the file at `dst` already exists, it will be truncated and its contents
+// overwritten.
+func MoveFileWithReplacement(src, dst string) error {
```

Comment:
It might be clearer to call this `CopyFileWithReplacement` to indicate that the bytes are being copied.  The word "move" often indicates a file relinking, not a copy.

Reply:
True, but the word "copy" indicates that the source is not moved. I took the naming from the unix utils which work the same way: ``mv`` will copy also when a file is on different partitions. 

To be honest, I only exported that name because the ``fs_test`` package needed access to it. :sweat_smile: